packages/api/src/index.ts

```diff
@@ -38,7 +38,15 @@ void (async () => {
           await entityManager.persistAndFlush(newUser);
         }
 
-        done(null, { profile, accessToken, refreshToken });
+        const guildIds = profile.guilds?.filter((guild) => guild.id === env.discordGuildId) ?? [];
+
+        if (guildIds.length > 0) {
```

Comment:
Just for future reference, anything greater than 0 is truth, so you really just need this:

```suggestion
        if (guildIds.length) {
```